src/uct/api/uct.h

```diff
@@ -712,7 +712,9 @@ enum {
     UCT_MD_FLAG_SOCKADDR   = UCS_BIT(7),  /**< MD support for client-server
                                                connection establishment via
                                                sockaddr */
-    UCT_MD_FLAG_INVALIDATE = UCS_BIT(8)   /**< MD supports memory invalidation */
+    UCT_MD_FLAG_INVALIDATE = UCS_BIT(8),  /**< MD supports memory invalidation */
+    UCT_MD_FLAG_DONT_CACHE = UCS_BIT(9)   /**< MD registrations should not be
```

Comment:
need to decide whether it will be ["don't cache"](https://github.com/openucx/ucx/pull/8339), "exact region" (this one, branches in fast-path, doubtful expedienсe),  or "no overlapping" (rcache complication, design)